Scripts/RemediationScripts/Remediate-SecureFTPDeploymentForAppServices.ps1

```diff
@@ -217,13 +237,14 @@ function Enable-SecureFTPDeploymentForAppServices
     # Setting up context for the current Subscription.
     $context = Set-AzContext -SubscriptionId $SubscriptionId -ErrorAction Stop
     
-    
+    if(-not($AutoRemediation))
+    {
     Write-Host "Subscription Name: [$($context.Subscription.Name)]"
     Write-Host "Subscription ID: [$($context.Subscription.SubscriptionId)]"
     Write-Host "Account Name: [$($context.Account.Id)]"
     Write-Host "Account Type: [$($context.Account.Type)]"
     Write-Host $([Constants]::SingleDashLine)
-    
+    }
```

Comment:
Line 279: Whenever the current script execution is ending, add a double dash line.

Here, add double dash line before 'return'. 

Similarly, add double dash line at other places as well.

Reply:
Added double dash line when execution is ending.